app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/ContainerFragment.kt

```diff
@@ -0,0 +1,61 @@
+package com.sun_asterisk.moviedb_50.screen.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentTransaction
+import com.sun_asterisk.moviedb_50.R
+
+class ContainerFragment : Fragment() {
+    private var startingPosition = 0;
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_container, container, false)
+        loadFragment(HomeFragment(), startingPosition)
+        return view
+    }
+    private fun loadFragment(
+        fragment: Fragment?,
+        position: Int
+    ): Boolean {
```

Comment:
hàm này tên không dài, không vượt quá 100 kí tự, bạn có thể viết chúng trên 1 dòng 
vì sao Fragment lại có thể null được nhỉ ? 
```suggestion
    private fun loadFragment(fragment: Fragment?, position: Int): Boolean {
```